tljh/requirements-base.txt

```diff
@@ -8,17 +8,14 @@
 # For JupyterHub 1.x SQLAlchemy below 2.0.0
 SQLAlchemy<2.0.0
 # JupyterHub + notebook package are base requirements for user environment
-jupyterhub==1.*
-notebook==6.*
+jupyterhub==3.1.*
+notebook==6.5.*
 # Install additional notebook frontends!
 jupyterlab==3.*
-nteract-on-jupyter==2.*
-# Install jupyterlab extensions from PyPI
+nteract-on-jupyter==2.1.*
 # nbgitpuller for easily pulling in Git repositories
-nbgitpuller==1.*
+nbgitpuller==1.1.*
```

Comment:
These are pinned to the minor version, I think we should stick with major versions here as well.

I can't say if that was an update part of the change you pushed now though Min, so now this is a bit mixxed up.

Reply:
I didn't change anything, I just rebased so it would run with the tests. I'm fine with shifting to just major versions.

Reply:
Now this only pins major versions, which is probably right when we aren't doing proper frozen environments